go-controller/pkg/util/ovs.go

```diff
@@ -839,7 +856,30 @@ func (t *NBTxn) Commit() (string, string, error) {
 	if len(t.txnArgs) == 0 {
 		return "", "", nil
 	}
-	allArgs := append(t.args, t.txnArgs...)
-	stdout, stderr, err := runOVNretry(runner.nbctlPath, t.env, allArgs...)
+
+	// assume a safe buffer of 10 args for a nbctl command
+	chunkSize := t.maxArgs - 10
+
+	stdout := bytes.NewBuffer([]byte{})
+	stderr := bytes.NewBuffer([]byte{})
+	var err error
+	// Chunk commands into acceptable sizes
```

Comment:
An alternative, and probably safer solution is to change Txn.Add to not allow more than a specific size and return a "max size" error. Then the caller can call commit and build a new transaction. @dcbw thoughts?

Reply:
Yeah, it's going to be tricky. I'm not sure we can automatically chunk it to a second invocation, because if you define a placeholder with `@` early on and then try to refer to it in a second chunk, I'll bet that fails.